src/Digdir.Domain.Dialogporten.Infrastructure/DomainEvents/Outbox/ConvertDomainEventsToOutboxMessagesInterceptor.cs

```diff
@@ -110,4 +101,23 @@ public override async ValueTask<int> SavedChangesAsync(SaveChangesCompletedEvent
 
         return await base.SavedChangesAsync(eventData, result, cancellationToken);
     }
+
+    private async ValueTask SendDialogUpdated(Guid dialogId, CancellationToken cancellationToken)
+    {
+        if (UpdateEventThrottleCache.TryGetValue(dialogId, out _))
+        {
+            return;
+        }
+
+        UpdateEventThrottleCache.Set(dialogId, true, TimeSpan.FromMilliseconds(50));
+
+        await _topicEventSender.SendAsync(
+            $"{Constants.DialogEventsTopic}{dialogId}",
+            new DialogEventPayload
+            {
+                Id = dialogId,
+                Type = DialogEventType.DialogUpdated
+            },
+            cancellationToken);
+    }
```

Comment:
Only one instance of the WebAPI will receive the REST request that generates these events though.

This throttle is meant to be "local", in the case that an update event *and* an activity created event is generated in the same transaction. The clients regard both of these as just one update, and does not need to be notified twice. 
(This would cause them to rapidly reload the same data twice) 

So there will be only one update event generated, and it will be pushed to Redis and all GraphQL containers that are listening on this DialogID will be triggered and end up notifying their subscribers once.

🌷 🐰